pyproject.toml

```diff
@@ -1,24 +1,24 @@
 [build-system]
 requires = [
-	"setuptools >= 65",
-	"Cython",
-	'numpy>=2.0.0rc1',
-	'setuptools_scm[toml] >= 8',
+    "setuptools >= 65",
+    "Cython",
+    "numpy >= 2.0.0rc1",
+    "setuptools_scm[toml] >= 8",
 ]
 build-backend = "setuptools.build_meta"
 
 [tool.black]
```

Comment:
I guess the black settings will go away if we stick with ruff

Reply:
ok, thanks for your remark, I removed the [tool.black] part from pyproject.toml